publish-to-npm.sh

```diff
@@ -23,6 +23,8 @@ fi
 
 set -x
 
+# Go to the package directory
```

Comment:
This script should probably still live in the dscc-gen directory. The top-level publish command should then delegate to each sub-package's custom publish script.

Reply:
I agree- done.